api/oc_helpers.c

```diff
@@ -288,27 +289,24 @@ oc_conv_hex_string_to_byte_array(const char *hex_str, size_t hex_str_len,
     return -1;
   }
 
-  size_t a = hex_str_len / 2.0 + 0.5;
+  long len = lround((double)hex_str_len / 2.0);
```

Comment:
is it possible to avoid double -> because operations over double is not cheap.

Reply:
yes we can get rid of it